src/components/Request/direct-request/views/Initial.direct.request.view.tsx

```
@@ -61,14 +61,19 @@ const DirectRequestInitialView = ({ username }: DirectRequestInitialViewProps) =
     }
 
     const isDisabled = useMemo(() => {
-        return !user?.username || !currentInputValue || (!!authUser?.user.userId ? !address : !recipient.address)
+        const parsedAmount = parseFloat(currentInputValue)
+        const isAmountInvalid = isNaN(parsedAmount) || parsedAmount <= 0
+        const isIdentityLogicMissing = !!authUser?.user.userId ? !address : !recipient.address
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)